src/server/api/downloads.js

```diff
@@ -90,15 +79,17 @@ const metadataTemplateCSVhandler = async (req, res) => {
 
 const doDownloadFromTerrain = (userID, accessToken, apiURL) => {
     logger.info(`TERRAIN ${userID} GET ${apiURL.href}`);
-    const requestOptions = {
+    let requestOptions = {
```

Comment:
Although this can remain a `let`, it can also stay as a `const`, since `requestOptions` isn't getting assigned to any other value (even though its contents are getting updated).

Reply:
Ah, right. Soft immutability, heh. Too many different programming languages in my brain, clearly :D